### PR TITLE
Upgrading to WalletConnect v1.0.0-beta

### PR DESCRIPTION
**Pull request by pedrouid**
Requires:

- [ ] balance-io/balance-common#74
- [ ] balance-io/balance-wallet#389

### CODE REVIEW

**Comment by ricburton** [comment]
🙌 Preview this pull request.

Built with commit e294378da085581c3288b206cd046c889aa138d4

https://deploy-preview-470--balance-manager.netlify.com

File: src/pages/index.js [hunk 1]
```diff
@@ -386,15 +370,16 @@ class Home extends Component {
...
-              <WalletConnectButton
-                color="walletconnect"
-                hoverColor="walletconnectHover"
-                activeColor="walletconnectActive"
-                onClick={this.onWalletConnectInit}
-                width={`100%`}
-              >
-                {lang.t('homepage.connect_walletconnect.button_mobile')}
-              </WalletConnectButton>
+              <Link to="/wallet">
```
Comment (jackjdleslie) [inline src/pages/index.js]: This'll need to be changed to `WalletConnectLink`, with the following style rules:

```js
const WalletConnectLink = styled(Link)`
  margin: 0 auto;
  max-width: 315px;
  width: 100%;
`;
```

Without this, on mobile the `TrustWalletLink` uses the full container width, whereas `Link` doesn't.

**Comment by pedrouid** [comment]
This can actually be merged right away. The Manager doesn't actually need the other two PR's to be merged first. cc @ricburton 

The was only on hold because the balance-wallet wasn't up-to-date with v1.0 but now it's not needed anymore

File: src/pages/index.js (hunk 1 above)
Comment (pedrouid) [reply src/pages/index.js]: Done! Just pushed this change. Thanks for the review @jackleslie! 🙌

**Review by ricburton — APPROVED** [review]
😇

**Pull request merged**